src/ensembl/src/content/app/browser/browserActions.ts

```diff
@@ -363,7 +363,6 @@ export const changeFocusObject = (
   }
 
   dispatch(updatePreviouslyViewedObjectsAndSave());
-
```

Comment:
could you revert this change please?

Reply:
yep, the removal of the line between dispatch and browserMessagingService.

In fact, could you run `git checkout dev <path to src/content/app/browser/browserActions.ts>`? It will restore the file to its state in dev branch; and you'll be able to commit the restored file.

Reply:
yup actually there was no change in that file LOL

Reply:
Done!